stylus/components/alerts.styl

```diff
@@ -27,6 +27,10 @@ $alert
         bottom auto
         text-align center
 
+$alert--modal
+    z-index $alert-index
+    bottom 0
```

Comment:
Because it should look like this:
﻿﻿﻿﻿﻿<img width="322" alt="screenshot 2018-08-03 16 09 58" src="https://user-images.githubusercontent.com/1280069/43647533-280766ba-9738-11e8-9d92-c197000c194d.png">

and on mobile, when there's no modal displayed, alerts are position above the Nav, so with `bottom: 48px;`.